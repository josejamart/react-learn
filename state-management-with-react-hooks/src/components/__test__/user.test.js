import React from "React";
import User from "../user";
import renderer from 'react-test-renderer';

describe('User -> Component', () => {

    it('Match snapshot', () => {
        const user = renderer
            .create(<User userName="User name" />)
            .toJSON();
        expect(user).toMatchSnapshot();
    });


})