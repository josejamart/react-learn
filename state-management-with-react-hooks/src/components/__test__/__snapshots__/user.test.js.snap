// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User -> Component Match snapshot 1`] = `
<span
  className="band-logo right"
>
  User name
</span>
`;
